panther_analysis_tool/main.py

```diff
@@ -1263,6 +1276,16 @@ def setup_parser() -> argparse.ArgumentParser:
         "type": strtobool,
         "help": "Meant for advanced users; allows skipping of extra keys from schema validation.",
     }
+    ignore_files_name = "--ignore-files"
+    ignore_files_arg: Dict[str, Any] = {
+        "required": False,
+        "dest": "ignored_files",
+        "nargs": "+",
+        "help": "Relative path to files in this project to be ignored by panther-analysis tool, "
+        + "space separated. Example ./foo.yaml ./bar/baz.yaml",
```

Comment:
Not a suggestion for change, the `+` is not actually required for string literals.